client/src/components/Navbar.jsx

```diff
@@ -0,0 +1,28 @@
+import navbarCSS from "./Navbar.module.css";
+import { Link, useMatch, useResolvedPath } from "react-router-dom";
+
+function Navbar() {
+   return (
+      <nav className={navbarCSS.nav}>
+         <Link to="/" className={navbarCSS["site-title"]}>
+            Weekend To-Do
+         </Link>
+         <ul>
+            <CustomLink to="/">Home</CustomLink>
+            <CustomLink to="/about">About</CustomLink>
+         </ul>
+      </nav>
+   );
+}
+
+function CustomLink({ to, children }) {
+   const resolvedPath = useResolvedPath(to);
+   const isActive = useMatch({ path: resolvedPath.pathname, end: true });
+
+   return (
+      <li className={`${isActive ? navbarCSS.active : ""}`}>
```

Comment:
you can use `classnames`
https://www.npmjs.com/package/classnames

```suggestion
      <li className={classnames({ [navbarCSS.active]: isActive })}>
```

Reply:
Oh cool, just added it here and in another comopnent,  thanks!